### ISSUE TITLE
Request: version tags

### ISSUE THREAD

**Issue by chinenual** [surge-synthesizer/tuning-library#49]
A non-functional request: could you tag the library with version markers (e.g., v1.0.0, v1.1.0, or some other tagname that makes sense to the project)?   This will make it easier for me to track and record which version of the C++ library my go port (https://github.com/chinenual/go-scala) corresponds to.

**Comment by baconpaul** [comment]
Oh sure. That's a good idea
How about I tag the version you ported to as 1.0 and I will tag the one I just pushed with a few fixes as 1.1?

I'll use the tag 'release_1.0.0" and so on as tags

Sound good?

**Comment by chinenual** [comment]
Works for me!   My tags won't necessarily be in sync with yours - I've already tagged 1.0.0, 1.1.0 (with some go-specific API changes) and 1.1.1 (to incorporate some unit tests you added a few weeks ago).  Whatever convention you use is fine with me - I can at least refer to it by name to document what version I'm theoretically in sync with at any point.

**Comment by baconpaul** [comment]
OK I pushed two tags

release_1.0.0 is the tag where I link our README to your project (so I presume it is what you ported). This is also the version surge 1.9 uses (basically)

release_1.1.0 is the version I just pushed with some fixed error messages, a new API for unmapped keys, some expanded tests, and a move to CMake on our side. This is the version surge XT currently uses and I think will use in end state

So closing this!
